sunday.tex

```diff
@@ -49,9 +49,9 @@ \section*{Sunday Sessions Descriptions}\label{sunday-descriptions}
   "`Mapathon, mapathon, mapathon!"'%
```

Comment:
Three different quotation marks here, are they needed at all?

Reply:
Oh, sorry, I converted most quotation marks to ```` and `''` which are the commands for proper English quotation marks in LaTeX, but it seems that I forgot some.